public/data/sooFeedData.json

```
@@ -0,0 +1,14 @@
+[
+  {
+    "num": 1,
+    "title": "MickyWanna_DC",
+    "title2": "/images/LeeSoo/feed1.jpg",
```

```suggestion
    "src": "/images/LeeSoo/feed1.jpg",
```  
이미지의 src니까 변수 명을 바꿔 주어도 좋을 것 같아요!